gammapy/data/event_list.py

```diff
@@ -74,6 +76,30 @@ def read(cls, filename, **kwargs):
         table = Table.read(filename, **kwargs)
         return cls(table=table)
 
+    def write(self, filename, gti=None, overwrite=False):
+        """Write the event list to a FITS file.
+
+        If a GTI object is provided, it is saved into
+        a second extension in the file.
+        """
+
+        filename = make_path(filename)
+
+        if gti is None: # save just the events
```

Comment:
I think this could be simplified to just appending the gti to the hdu list if given instead of duplicating the writing code

Reply:
```
hdus = [fits.PrimaryHDU(), fits.BinTableHDU(self.table, name='EVENTS)]

if gti is not None:
    hdus.append(fits.BinTableHDU(gti.table, name="GTI")
    
fits.HDUList(hdus).writeto(filename, overwrite=True)
```

Reply:
oh this comment didn't appear until after I pushed - but my implementation is essentially the same